webpack.config.js

```diff
@@ -160,22 +163,45 @@ module.exports = ( env ) => {
     }
   }
 
+  function getCustomIfUtils( { ifDev, ifProd, ifSite } = {} ) {
 
-  function ifProdOrSite( returnValue = true ) {
+    return {
+
```

Comment:
Yup yup. That's why I wrote "next time". Formatter had a bug with ending spaces in curly braces which I would like to also removr, the code looks wierd and long with all those spaces. We will discuss is in person for sure. Thanks!